spec/index.html

```diff
@@ -197,22 +192,103 @@
         <pre class="idl">
         enum AutocompleteType { "none", "inline", "list", "both" };
         enum CheckedType { "false", "mixed", "true" };
+        enum CurrentType { "page", "step", "location", "date", "time", "true", "false" };
         enum InvalidType { "false", "true", "grammar", "spelling" };
         enum LiveType { "off", "polite", "assertive" };
         enum LiveRelevantType { "additions", "additions text", "all", "removals", "text" };
         enum OrientationType { "horizontal", "vertical" };
+        enum RoleType {
+          "alert",
+          "alertdialog",
+          "application",
+          "article",
+          "banner",
+          "button",
+          "cell",
+          "checkbox",
+          "columnheader",
+          "combobox",
+          "complementary",
+          "contentinfo",
+          "definition",
+          "dialog",
+          "directory",
+          "document",
+          "feed",
+          "figure",
+          "form",
+          "grid",
+          "gridcell",
+          "group",
+          "heading",
+          "img",
```

Comment:
Hmmm, I'm not so sure we want to use a different lexicon than ARIA here.
Is it worth it to confuse authors and force authors who try to make glue
code to translate things?